app/serializers/rents_serializers.rb

```diff
@@ -0,0 +1,5 @@
+class RentsSerializer < ActiveModel::Serializer
```

Comment:
Rename to RentSerializer and remember to rename file

Reply:
Done